packages/expo-dev-launcher/plugin/src/resolvePackageRootFolder.ts

```diff
@@ -0,0 +1,13 @@
+import findUp from 'find-up';
+import * as path from 'path';
+import resolveFrom from 'resolve-from';
+
+export function resolvePackageRootFolder(fromDirectory: string, moduleId: string): string | null {
+  const resolved = resolveFrom.silent(fromDirectory, moduleId);
+  if (!resolved) return null;
```

Comment:
```suggestion
  if (!resolved) {
  	return null;
  }
```

Reply:
This file is actually copied directly from https://github.com/expo/expo-cli/blob/master/packages/config-plugins/src/utils/resolvePackageRootFolder.ts -- I'd like to leave it the same so that it's easier to spot & sync any changes.

Reply:
Never mind, ended up removing this file entirely 😄